### DISCUSSION TITLE
Build option for Win11 ARM

### DISCUSSION THREAD

**Discussion by se** [FreeRDP/FreeRDP#8137] [Q&A]
Hi, is there any possibility of building the project in Win 11 ARM? I am using VS 2022 and getting the following error message about the target arch.

`
    C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Current\Bin\arm64\Microsoft.Common.CurrentVersion.targets(823,5): error : The BaseOutputPath/OutputPath property is not set for project 'VCTargetsPath.vcxproj'.  Please check to make sure that you have specified a valid combination of Configuration and Platform for this project.  Configuration='Debug'  Platform='ARM64'.  You may be seeing this message because you are trying to build a project without a solution file, and have specified a non-default Configuration or Platform that doesn't exist for this project. 
`

**Comment by akallabeth** [comment]
[FreeRDP compile manual](https://github.com/FreeRDP/FreeRDP/wiki/Compilation)
